src/database/utilities/preferenceConstants.js

```diff
@@ -16,6 +16,7 @@ export const PREFERENCE_KEYS = {
   HIDE_SNAPSHOT_COLUMN: 'usesHideSnapshotColumn',
   CAN_EDIT_PATIENTS_FROM_ANY_STORE: 'can_edit_patients_from_any_store',
   CAN_VIEW_ALL_PATIENTS_HISTORY: 'can_view_all_patients_history',
+  NEW_PATIENTS_VISIBLE_THIS_STORE_ONLY: 'new_patients_visible_in_this_store_only',
```

Comment:
Based this on https://github.com/sussol/msupply/issues/9254#issuecomment-888164810 - I'm not too sure if this is safe enough to use 🤔